tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/NewLiferayComponentServicePossibleValuesService.java

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.project.core.modules;
+
+import com.liferay.ide.project.core.ProjectCore;
+import com.liferay.ide.server.core.portal.PortalServer;
+
+import java.util.Arrays;
+import java.util.Set;
+
+import org.eclipse.sapphire.FilteredListener;
+import org.eclipse.sapphire.Listener;
+import org.eclipse.sapphire.PossibleValuesService;
+import org.eclipse.sapphire.PropertyContentEvent;
+import org.eclipse.sapphire.Value;
+import org.eclipse.sapphire.modeling.Status;
+import org.eclipse.wst.server.core.IServer;
+import org.eclipse.wst.server.core.ServerCore;
+
+/**
+ * @author Simon Jiang
+ * @author Lovett Li
+ */
+public class NewLiferayComponentServicePossibleValuesService extends PossibleValuesService
```

Comment:
This is not needed, since based on the Component template we know what the service class should be.